pkg/LibTorch.GPU.Redist/LibTorch.GPU.Redist.nupkgproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <Authors></Authors>
+    <Copyright>Copyright (c) Microsoft Corporation</Copyright>
+    <Owners>Microsoft Corporation</Owners>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>$(MSBuildProjectName) contains the PyTorch C++ frontend LibTorch library version $(LibTorchVersion) 
+    for GPU with Cuda redistributed as a NuGet package. </PackageDescription>
```

Comment:
We should probably specify that this is using LibTorch with CUDA 9.0

Reply:
Yes. Also can we change the name of the redist? Something like LibTorch.GPU.CUDA.9.0.Redist? So that eventually we can create a CUDA 10 as well

Reply:
Yes I think it's a good idea.